engines/glk/scott/scott.cpp

```diff
@@ -408,8 +408,8 @@ void Scott::outputNumber(int a) {
 }
 
 void Scott::look(void) {
-	const uint32 *const ExitNames[6] = {
-		_("North").c_str(), _("South").c_str(), _("East").c_str(), _("West").c_str(), _("Up").c_str(), _("Down").c_str()
+	const Common::U32String ExitNames[6] = {
+		_("North"), _("South"), _("East"), _("West"), _("Up"), _("Down")
 	};
```

Comment:
A possibly better alternative to this would have been to keep using an array of `char *` and only translate them when actually used (note the use of `_s()` instead of `_()` in the array definition, and then `_()` being used when calling `display()`):
```c++
const char *const ExitNames[6] = {
		_s("North"), _s("South"), _s("East"), _s("West"), _s("Up"), _s("Down")
	};
...
	display(_topWindow, Common::U32String("%S"), _(ExitNames[ct]).c_str());
```

Reply:
That is indeed a better alternative. It also closely matches the original code. Have changed it.